src/views/Login.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {Link} from "react-router-dom";
+
+const login = () => {
+    return (
+        <div className="fullScreenLayout">
+            <div className="col-md-4">
+                <h1>
+                    Vpiši se <i className='fas fa-user-shield'/>
+                </h1>
+                <form
```

Comment:
The register form has the attribute 'required' on all of its components. Why is this missing here?